cpu/atmega1281/include/cpu_conf.h

```diff
@@ -29,17 +29,22 @@ extern "C" {
  * @name Kernel configuration
  *
  * Since printf seems to get memory allocated by the linker/avr-libc the stack
- * size tested sucessfully even with pretty small stacks.k
+ * size tested sucessfully even with pretty small stacks.
  * @{
  */
 #define THREAD_EXTRA_STACKSIZE_PRINTF    (128)
 
 #ifndef THREAD_STACKSIZE_DEFAULT
-#define THREAD_STACKSIZE_DEFAULT   (256)
+#   define THREAD_STACKSIZE_DEFAULT   (256)
```

Comment:
@OlegHahm did you intend to add these tabs? I'd vote to remove them and merge then